src/Common/src/System/Net/CookieParser.cs

```diff
@@ -623,7 +632,7 @@ internal Cookie Get()
                                         else
                                         {
                                             // This cookie will be rejected
-                                            cookie.InternalSetName(string.Empty);
+                                            SetCookieName(cookie, string.Empty);
```

Comment:
Isn't this going to just be a nop that throws an exception and eats it?  Why call it at all?